app/pandas.ipynb

```diff
@@ -458,36 +1195,78 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
     "s = pd.Series([\"numpy\", \"pandas\", \"matplotlib\"])"
    ],
    "metadata": {
-    "collapsed": false
-   }
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2024-10-23T17:27:38.913196Z",
+     "start_time": "2024-10-23T17:27:38.908672Z"
+    }
+   },
+   "outputs": [],
+   "execution_count": 50
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# make all words in uppercase"
+    "s = s.apply(lambda x: x.upper())\n",
+    "s"
    ],
    "metadata": {
-    "collapsed": false
-   }
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2024-10-23T17:29:26.383891Z",
+     "start_time": "2024-10-23T17:29:26.376686Z"
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0         NUMPY\n",
+       "1        PANDAS\n",
+       "2    MATPLOTLIB\n",
+       "dtype: object"
+      ]
+     },
+     "execution_count": 53,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "execution_count": 53
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# get length of each word"
+    "s = s.apply(lambda x: len(x))\n",
+    "s"
```

Comment:
Ensure that the operation to get the length of each word does not modify the original Series 's'. Consider creating a new Series or using a method that doesn't alter 's' directly.